packages/blocks/src/api/index.js

```diff
@@ -1,3 +1,9 @@
+// The Block Type is the most important concept within the block API. It defines
```

Comment:
Should "Block Type" be capitalized or not? It isn't elsewhere in these comments.